be/src/storage/rowset_column_update_state.h

```diff
@@ -51,14 +51,17 @@ struct RowsetSegmentId {
     uint32_t segment_id;
 };
 
+// from source rowid -> upt rowid
+using RowidPairs = std::pair<uint32_t, uint32_t>;
+
 struct ColumnPartialUpdateState {
     bool inited = false;
     // Maintains the mapping of each row of data in the update segment to source row
     std::vector<uint64_t> src_rss_rowids;
     // this state been generated by this version
     EditVersion read_version;
-    // Maintains the mapping of source row to update segment's row
-    std::map<uint64_t, uint32_t> rss_rowid_to_update_rowid;
+    // rssid -> source rowid -> update file's rowid
+    std::map<uint32_t, std::vector<RowidPairs>> rss_rowid_to_update_rowid;
```

Comment:
Need to explain in the comment the reason that it can save a lot of memory usage.

Reply:
done.